string name
Location location
---
bool result